README.md

```diff
@@ -4,7 +4,7 @@ README
 ======
 This project (and its dependencies) contains the EuroPython website source code.
 
-The code is used for the EuroPython 2019 website.
```

Comment:
Weird that this change is shown here. The latest version of the file already has this change https://github.com/EuroPython/epcon/blob/dev/ep2020/README.md